soc/arm/silabs_exx32/Kconfig

```diff
@@ -326,4 +326,11 @@ config SOC_GECKO_USE_RAIL
 	  hardware. This option enable the proper set of features to allow to properly compile
 	  with the RAIL blob.
 
+config SOC_GECKO_USE_RAIL_API
+	bool "Use RAIL (Radio Abstraction Interface Layer) API Functions"
+	help
+	  RAIL (Radio Abstraction Interface Layer) with this enabled
+	  the APIs can be used for managing the radio interface of the chip.
+	  Requires SOC_GECKO_USE_RAIL to be enabled.
```

Comment:
`depends on SOC_GECKO_USE_RAIL`?

Reply:
Hi, the previous PR prepared RAIL to be able to be used correctly for BLE sample applications.
Now we are working on that with this update, the missing files will be also included and added to the source, that are needed for the Radio to be able to use the Radio Configurator generated c and h files for proprietary usage.

Reply:
> Please also see this PR for silab_hal: [zephyrproject-rtos/hal_silabs#39](https://github.com/zephyrproject-rtos/hal_silabs/pull/39)

Could you update the manifest file, so it points to this PR for hal_silabs?

Reply:
I would like to clarify if the PR message update is enough or should I also update the commit message as well?

Reply:
What @mrfuchs meant is that you should update the revision of the `hal_silabs` module in `west.yml` to point to your PR to the `hal_silabs` repository.

https://docs.zephyrproject.org/latest/develop/modules.html#submitting-changes-to-modules

Reply:
Thank you for pointing that out, I have updated the PR accordingly.